meter/rct.go

```diff
@@ -0,0 +1,156 @@
+package meter
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/evcc-io/evcc/api"
+	"github.com/evcc-io/evcc/util"
+
+	"github.com/mlnoga/rct"
+)
+
+/*
+This meter supports devices implementing the RCT communication protocol, e.g. the RCT PS 6.0 with / without battery.
+
+** Usages **
+The following usages are supported:
+- grid    ... for reading the power imported or exported to the grid
+- pv      ... for reading the power produced by the pv
+- battery ... for reading the power imported or exported to the battery
+
+** Example configuration **
+meters:
+- name: GridMeter
+  type: rct
+  host: 192.168.1.23
+  cache: 2s
+  usage: grid
+- name: PvMeter
+  type: rct
+  host: 192.168.1.23
+  cache: 2s
+  usage: pv
+- name: BatteryMeter
+  type: rct
+  host: 192.168.1.23
+  cache: 2s
+  usage: battery
+
+** Limitations **
```

Comment:
Lgess reference

Reply:
What does this mean? Would you like a reference, based on less.go?

Reply:
I've removed the reference to lgess- copy/paste leftover.